### PR TITLE
Update unconference

### PR DESCRIPTION
**Pull request by grimrose**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -574,15 +574,15 @@ <h3 style="padding: 1em; margin-top: 2em">2月26日 (日曜日)</h3>
...
 								<td class="roomB">
 									<span class="session_id">S-11A</span>
 									{% include schedule_ja.html id="YukiNakayama_2" %}
 								</td>
-								<td class="roomC" rowspan="2">TBD</td>
-								<td class="roomD" rowspan="2">TBD</td>
-								<td class="roomE" rowspan="2">TBD</td>
-								<td class="roomF" rowspan="2">TBD</td>
+								<td class="roomC" rowspan="2">Finagle session - Matsumoto</td>
+								<td class="roomD" rowspan="2">Lens の基本 - Aoyama</td>
```
Comment (yohei1126) [inline index.html]: Introduction to Lens - Aoyama

File: index.html [hunk 2]
```diff
@@ -599,15 +599,15 @@ <h3 style="padding: 1em; margin-top: 2em">2月26日 (日曜日)</h3>
...
 								<th>11:50 - 12:05</th>
-								<td class="roomA" rowspan="2">TBD</td>
+								<td class="roomA" rowspan="2">Recursion schemes in Scala - Pawel</td>
 								<td class="roomB">
 									<span class="session_id">S-12A</span>
 									{% include schedule_ja.html id="IkuoMatsumura_1" %}
 								</td>
-								<td class="roomC" rowspan="2">TBD</td>
-								<td class="roomD" rowspan="2">TBD</td>
+								<td class="roomC" rowspan="2">採用担当者、CTO ぶっちゃけ座談会 - Oe</td>
```
Comment (yohei1126) [inline index.html]: A panel session on carrer by recruiters, CTOs - Oe

File: index.html [hunk 3]
```diff
@@ -599,15 +599,15 @@ <h3 style="padding: 1em; margin-top: 2em">2月26日 (日曜日)</h3>
...
+								<td class="roomD" rowspan="2">Performance tuning for Scala and/or JVM - Amaya</td>
 								<td class="roomE" rowspan="2">TBD</td>
-								<td class="roomF" rowspan="2">TBD</td>
+								<td class="roomF" rowspan="2">Scala ビッグデータで困っていること共有しましょう - Kimura</td>
```
Comment (yohei1126) [inline index.html]: Let's share your issues on big data using Scala - Kimura

File: index.html [hunk 4]
```diff
@@ -624,31 +624,31 @@ <h3 style="padding: 1em; margin-top: 2em">2月26日 (日曜日)</h3>
...
 									<span class="session_id">B-14</span>
 									{% include schedule_ja.html id="VladimirKostyukov_1" %}
 								</td>
-								<td class="roomC">TBD</td>
-								<td class="roomD">TBD</td>
+								<td class="roomC">Poor man's type classes revisited - Sano</td>
+								<td class="roomD">Scala introduction for Java developers - @seratch</td>
 								<td class="roomE">TBD</td>
-								<td class="roomF">TBD</td>
+								<td class="roomF">かとじゅんさんを囲んで更に外に話せないことを聞きまくる会 - Ishikawa, Kato</td>
```
Comment (yohei1126) [inline index.html]: Ask Me Anything with Jun Kato - Ishikawa, Kato

File: index.html [hunk 5]
```diff
@@ -658,14 +658,14 @@ <h3 style="padding: 1em; margin-top: 2em">2月26日 (日曜日)</h3>
...
+									Designing APIs for binary compatibility over time - Sebastien
 								</td>
 								<td class="roomB">
 									<span class="session_id">B-15</span>
 									{% include schedule_ja.html id="MikhailMutcianko_1" %}
 								</td>
-								<td class="roomC">TBD</td>
-								<td class="roomD">TBD</td>
+								<td class="roomC">What is composability in Scala? - りりろじ</td>
+								<td class="roomD">Play2+DDDで挫折した話、みんなならどうする？ - Abe</td>
```
Comment (yohei1126) [inline index.html]: I gave up Play + DDD once, how you can solve this? - Abe

File: index_en.html [hunk 6]
```diff
@@ -651,14 +651,14 @@ <h3 style="padding: 1em; margin-top: 2em">Sunday, February 26th</h3>
...
+									Designing APIs for binary compatibility over time - Sebastien
 								</td>
 								<td class="roomB">
 									<span class="session_id">B-15</span>
 									{% include schedule_en.html id="MikhailMutcianko_1" %}
 								</td>
-								<td class="roomC">TBD</td>
-								<td class="roomD">TBD</td>
+								<td class="roomC">What is composability in Scala? - りりろじ</td>
+								<td class="roomD">Play2+DDDで挫折した話、みんなならどうする？ - Abe</td>
```
Comment (yohei1126) [inline index_en.html]: I gave up Play + DDD once, how you can solve this? - Abe

File: index_en.html [hunk 7]
```diff
@@ -617,31 +617,31 @@ <h3 style="padding: 1em; margin-top: 2em">Sunday, February 26th</h3>
...
 									<span class="session_id">B-14</span>
 									{% include schedule_en.html id="VladimirKostyukov_1" %}
 								</td>
-								<td class="roomC">TBD</td>
-								<td class="roomD">TBD</td>
+								<td class="roomC">Poor man's type classes revisited - Sano</td>
+								<td class="roomD">Scala introduction for Java developers - @seratch</td>
 								<td class="roomE">TBD</td>
-								<td class="roomF">TBD</td>
+								<td class="roomF">かとじゅんさんを囲んで更に外に話せないことを聞きまくる会 - Ishikawa, Kato</td>
```
Comment (yohei1126) [inline index_en.html]: Ask Me Anything with Jun Kato at ChatWork - Ishikawa, Kato

File: index_en.html [hunk 8]
```diff
@@ -592,15 +592,15 @@ <h3 style="padding: 1em; margin-top: 2em">Sunday, February 26th</h3>
...
 								<th>11:50 - 12:05</th>
-								<td class="roomA" rowspan="2">TBD</td>
+								<td class="roomA" rowspan="2">Recursion schemes in Scala - Pawel</td>
 								<td class="roomB">
 									<span class="session_id">S-12A</span>
 									{% include schedule_en.html id="IkuoMatsumura_1" %}
 								</td>
-								<td class="roomC" rowspan="2">TBD</td>
-								<td class="roomD" rowspan="2">TBD</td>
+								<td class="roomC" rowspan="2">採用担当者、CTO ぶっちゃけ座談会 - Oe</td>
```
Comment (yohei1126) [inline index_en.html]: A panel session by recruiters, CTOs - Oe

File: index_en.html [hunk 9]
```diff
@@ -592,15 +592,15 @@ <h3 style="padding: 1em; margin-top: 2em">Sunday, February 26th</h3>
...
+								<td class="roomD" rowspan="2">Performance tuning for Scala and/or JVM - Amaya</td>
 								<td class="roomE" rowspan="2">TBD</td>
-								<td class="roomF" rowspan="2">TBD</td>
+								<td class="roomF" rowspan="2">Scala ビッグデータで困っていること共有しましょう - Kimura</td>
```
Comment (yohei1126) [inline index_en.html]: Let's share your issues on big data using Scala - Kimura

File: index_en.html [hunk 10]
```diff
@@ -567,15 +567,15 @@ <h3 style="padding: 1em; margin-top: 2em">Sunday, February 26th</h3>
...
 								<td class="roomB">
 									<span class="session_id">S-11A</span>
 									{% include schedule_en.html id="YukiNakayama_2" %}
 								</td>
-								<td class="roomC" rowspan="2">TBD</td>
-								<td class="roomD" rowspan="2">TBD</td>
-								<td class="roomE" rowspan="2">TBD</td>
-								<td class="roomF" rowspan="2">TBD</td>
+								<td class="roomC" rowspan="2">Finagle session - Matsumoto</td>
+								<td class="roomD" rowspan="2">Lens の基本 - Aoyama</td>
```
Comment (yohei1126) [inline index_en.html]: Introduction to Lens - Aoyama

File: index_en.html [hunk 11]
```diff
@@ -617,31 +617,31 @@ <h3 style="padding: 1em; margin-top: 2em">Sunday, February 26th</h3>
...
 								<td class="roomB">
 									<span class="session_id">B-13</span>
 									{% include schedule_en.html id="RegisBlanc_1" %}
 								</td>
-								<td class="roomC">TBD</td>
-								<td class="roomD">TBD</td>
-								<td class="roomE">TBD</td>
-								<td class="roomF">TBD</td>
+								<td class="roomC">Safety beyond Types	Scala エンジニアのための英会話教室 - Chrisたぁ</td>
+								<td class="roomD">ゆるふわ Scala 女子会 - Kayano</td>
```
Comment (yohei1126) [inline index_en.html]: Girls talk on Scala (Guys are also welcomed) - Kayono

File: index_en.html [hunk 12]
```diff
@@ -617,31 +617,31 @@ <h3 style="padding: 1em; margin-top: 2em">Sunday, February 26th</h3>
...
+								<td class="roomA">RealTime network intrusion detection using deep learning - Adam</td>
 								<td class="roomB">
 									<span class="session_id">B-13</span>
 									{% include schedule_en.html id="RegisBlanc_1" %}
 								</td>
-								<td class="roomC">TBD</td>
-								<td class="roomD">TBD</td>
-								<td class="roomE">TBD</td>
-								<td class="roomF">TBD</td>
+								<td class="roomC">Safety beyond Types	Scala エンジニアのための英会話教室 - Chrisたぁ</td>
```
Comment (yohei1126) [inline index_en.html]: English conversation lesson for Scala engineers

File: index_en.html [hunk 13]
```diff
@@ -617,31 +617,31 @@ <h3 style="padding: 1em; margin-top: 2em">Sunday, February 26th</h3>
...
+								<td class="roomD">ゆるふわ Scala 女子会 - Kayano</td>
+								<td class="roomE">sbt clinic - Yokota</td>
+								<td class="roomF">エンジニアのキャリア相談会 - Ishibashi</td>
```
Comment (yohei1126) [inline index_en.html]: Career counseling for engineers - Ishibashi

File: index.html [hunk 14]
```diff
@@ -624,31 +624,31 @@ <h3 style="padding: 1em; margin-top: 2em">2月26日 (日曜日)</h3>
...
+								<td class="roomA">RealTime network intrusion detection using deep learning - Adam</td>
 								<td class="roomB">
 									<span class="session_id">B-13</span>
 									{% include schedule_ja.html id="RegisBlanc_1" %}
 								</td>
-								<td class="roomC">TBD</td>
-								<td class="roomD">TBD</td>
-								<td class="roomE">TBD</td>
-								<td class="roomF">TBD</td>
+								<td class="roomC">Scala エンジニアのための英会話教室 - Chrisたぁ</td>
```
Comment (xuwei-k) [inline index.html:632]: `Chrisたぁ` とは

Comment (grimrose) [reply index.html:632]: Sorry. ref #455

**Pull request merged**